Misc/NEWS.d/next/Library/2022-07-25-15-45-06.gh-issue-95231.i807-g.rst

```diff
@@ -0,0 +1,2 @@
+If kernel fips is enabled, we get permission error upon doing import crypt.
+So, if kernel fips is enabled, disable the unallowed hashing methods.
```

Comment:
Please update the NEWS entry to more accurately reflect the change. You may mention FIPS as a case where EPERM is triggered.